src/components/SnippetAreaToolbar.jsx

```diff
@@ -61,33 +52,40 @@ const SnippetAreaToolbar = (props) => {
       <TextField
         hintText="Snippet Name"
         id="titleField"
+        fullWidth
         onChange={onTitleChange}
-        style={styles.toolbarField}
         value={title}
       />
-      <LanguageSelector
-        disabled={readOnly}
-        language={language}
-        onChange={onLanguageChange}
-        style={styles.toolbarField}
-      />
-      <div>
-        <LockButton
-          onClick={onLockClick}
-          readOnly={readOnly}
-          style={styles.buttons}
-        />
-        <SaveMenu
-          canSave={canSave}
-          enabled={saveEnabled}
-          id="saveMenu"
-          onSaveAsClick={onSaveAsClick}
-          onSaveClick={onSaveClick}
-          style={styles.buttons}
-          orgs={orgs}
-          onOrgChanged={onOrgChanged}
-          selectedOrg={selectedOrg}
+      <div
+        style={{
```

Comment:
For consistency's sake, this style object should be moved into the `styles` constant above.